test/integration/test_minitest.rb

```diff
@@ -20,4 +20,21 @@ def test_verify
 Expected 1 times.
     MESSAGE
   end
+
+  test("assert_received increments assertions count") do
+    test_class = Class.new(Minitest::Test) do
+      def test_assert_received
+        object = Object.new
+        stub(object).hello { "Hello!" }
+        object.hello
+        assert_received(object) do |expect|
+          expect.hello
+        end
+      end
+    end
+
+    result = test_class.new(:test_assert_received).run
+    assert_empty result.failures
```

Comment:
Do we need this? (Is it important in this test?)

Reply:
True, removed.